schema/admins/defs.yaml

```diff
@@ -73,9 +73,15 @@ description: Common schema definitions for admins theme
     isoCountryCodeAlpha2:
       description: ISO 3166-1 alpha-2 country code.
       type: string
+      minLength: 2
+      maxLength: 2  
+      pattern: ^[A-Z]{2}$
     isoSubCountryCode:
       description: ISO-3166-2 Country subdivision code.
       type: string
+      minLength: 4
+      maxLength: 6

```

Comment:
Do you need the min/max length in both if the regex requires them?

Reply:
As an answer to your question: no.
But I would keep the min length, cause it makes the schema easier to read (granted here it is not a complex regex).
Also if you would have a validator who doesn't have proper regex support, the length fields are a good fall back mechanism.

Reply:
I tend to agree with @TristanDiet-TomTom.